1.8/translations/translations.md

```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
 #### Finishing Touches
 
 Go to **User CP > Edit Options**. You should see the translation in the **Other Options** menu. Try out the new translation and make sure that everything works.
+
+### Default forum translation
+
+If you want your new translation to be default for all new users and guests you can change it by going to **Admin CP > Configuration > Settings > General Configuration > Default Language**.
+This would make this the default translation for all new users and guests.
+
+**Please note:** Changing this in Admin CP will not force new default translation to existing users on board. They need to change it manually in their profiles.
+
+#### Default translation for Admin CP
+
+You can set the default Admin CP language to be different than default forums translation.
+To do that you should go to **Admin CP > Home > Preferences > Admin Control Panel Language**
+
+Note that you have options **Default** and languages installed on your board. Option Default will make default Admin CP translation same as board translation.
```

Comment:
This little section should be:
You can set the default Admin CP language to be different than the default forums translation.
To do that you should go to **Admin CP > Home > Preferences > Admin Control Panel Language**

Note that you have the options **Default** and languages installed on your board. Option Default will make the default Admin CP translation the same as the forum translation.